### PR TITLE
Add code documentation

### PR DESCRIPTION
**Pull request by CaselIT**
Add documentation to the code

While adding the docs I've notices this strangeness in https://github.com/vytas7/falcon-sqla/issues/3
I've not added this to the docs, but maybe it could be appropriate to add it?

### CODE REVIEW

**Comment by CaselIT** [comment]
I'm not sure why pypy coverage now fails, since I haven't changed any code

**Comment by vytas7** [comment]
Thanks, I'll take a look!

Not sure about that coverage miss either. `-96` looks very odd, or at least I don't recognize it, [coverage normally either list lines, or branches with `->`](https://coverage.readthedocs.io/en/coverage-5.0.3/cmd.html#reporting).

**Comment by CaselIT** [comment]
indeed, line 96 is black..

**Comment by vytas7** [comment]
Maybe we can try asking a question on the coverage bugtracker...

I'll add some cosmetic edit, and keep the failed build for the reference.

**Comment by vytas7** [comment]
I've reported this on the `coveragepy` bug tracker.
Meanwhile, I'll check if Travis supports the newest PyPy3 (7.3.0), and if that could fix the issue.

**Comment by vytas7** [comment]
7.3.0 is unsupported, but I got an upgrade to 7.2.0, let's see...

**Comment by CaselIT** [comment]
to the problem was the pypy version. how strange

**Comment by CaselIT** [comment]
friendly ping

File: falcon_sqla/__init__.py [hunk 1]
```diff
@@ -14,6 +14,6 @@
 
 from .manager import Manager
 
-__all__ = [
+__all__ = (
```
Comment (vytas7) [inline falcon_sqla/__init__.py]: Do we really need to change this? I'm also a bit of control freak myself loving to make everything immutable (what can be made immutable), however, [Python doc examples do use lists](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package).

So IMHO better to stick to that.

**Review by vytas7 — APPROVED** [review]
Thanks a lot for submitting this!
I'll probably need to polish this to my liking, but we'll need to actually build the docs first anyway.

**Comment by CaselIT** [comment]
Cheers 🍻

**Pull request merged**